### ISSUE TITLE
Is iTerm2 supported?

### ISSUE THREAD

**Issue by mattfordham** [achiurizo/terminitor#3]
I added the OSX core, but I still get a "No valid core was found!. Exiting..." message. Thanks!

**Comment by achiurizo** [comment]
I'm still adding in iTerm back in. I'll post back here shortly when I'm finished.

**Comment by mattfordham** [comment]
Great, thanks!

**Comment by achiurizo** [comment]
@mattfordham I've added support for [iTerm here](https://github.com/achiu/consular-iterm). Let me know if this works for you. Thanks!

**Comment by mattfordham** [comment]
Thanks! I installed and gave it a spin. Unfortunately, there seems to be an issue with tabs and panes. Running the following, as outlined in the README, opens a new tab but doesn't create any panes. I am using the latest iTerm2: Build 1.0.0.20111020.

```
tab do
  pane "gitx"    # first pane
    pane do      # second pane level => horizontal split
      run "irb"
    end
  pane 'ls'      # first pane level => vertical split
end
```

**Comment by achiurizo** [comment]
@mattfordham yeah I'm aware of this issue. I should update the README. to work around it, don't wrap the `pane` calls in `tab`. you could do:

``` ruby
pane do
  pane 'hey'
end
pane do
  pane 'ls'
end
```

and that would work. Alternatively, you can use `window` instead of `tab` to wrap the panes as well. I'll see if i can fix that issue though.
